### PR TITLE
Feature/recaptcha v3 keeping v2

### PR DESCRIPTION
**Pull request by cazapatamar**
## 🛠️ Changes
- reCAPTCHA v2 and v3 simultaneously in the signup and contact forms

## 📝 Associated issues
#424 

## 🤔 Considerations
Obtain the keys from the Google reCAPTCHA website for now. When GitLab becomes available, I will update the new environment variable names and values there.

### CODE REVIEW

File: app/controllers/registrations_controller.rb [hunk 1]
```diff
@@ -3,10 +3,15 @@ class RegistrationsController < Devise::RegistrationsController
 
   private
     def check_captcha
-      unless verify_recaptcha
+      recaptcha_v3_valid = verify_recaptcha(action: 'registration', minimum_score: 0.8, secret_key: ENV['RECAPTCHA_SECRET_KEY_V3'])
+		  recaptcha_v2_valid = verify_recaptcha unless recaptcha_v3_valid
```
Comment (erikasv) [inline app/controllers/registrations_controller.rb:7]: wring indentation

File: app/controllers/home_controller.rb [hunk 2]
```diff
@@ -28,29 +28,36 @@ def contact_us
...
+			end
+			if !@contact_message.errors.messages[:content].blank?
+				errores << "Mensaje | "
+			end
+			redirect_to home_contact_us_path, :flash => { :error => errores }
 		end
 	end
 
+
+
```
Comment (erikasv) [inline app/controllers/home_controller.rb:60]: remove these lines

File: app/controllers/registrations_controller.rb [hunk 3]
```diff
@@ -1,17 +1,21 @@
 class RegistrationsController < Devise::RegistrationsController
-  prepend_before_action :check_captcha, only: [:create]
+  	prepend_before_action :check_captcha, only: [:create]
```
Comment (erikasv) [inline app/controllers/registrations_controller.rb:2]: fix indentation

**Review by erikasv — CHANGES_REQUESTED** [review]
lgtm just some small adjustments

**Review by ManuelStardust — APPROVED** [review]

**Pull request merged**